### PR TITLE
Create workflow.yml

### PR DESCRIPTION
**Pull request by soonhongooi**

### CODE REVIEW

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        node-version: [10.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install -g yarn
+      - name: yarn install, build, test and lint
```
Comment (sjunejo) [inline .github/workflows/workflow.yml]: any reason `yarn` `yarn build` `yarn test` and `yarn lint` can't be separate steps?

Comment (soonhongooi) [reply .github/workflows/workflow.yml]: is it best practice to split them? does it make it slower?
I'm basing my approach via these repo

https://github.com/codyogden/killedbygoogle/blob/master/.github/workflows/build.yml

Comment (sjunejo) [reply .github/workflows/workflow.yml]: I think it's easier to identify issues in the console if the commands are split up into steps.

Atm this is what I see in the UI for the most recent commit - if e.g. `yarn test` failed the build, I'd have to scroll through the text to find out.

image

Comment (soonhongooi) [reply .github/workflows/workflow.yml]: I've split the run commands

**Pull request merged**